showcase/app/services/shw-theming.ts

```diff
@@ -0,0 +1,102 @@
+import type Owner from '@ember/owner';
+import { tracked } from '@glimmer/tracking';
+import { inject as service } from '@ember/service';
+
+import HdsThemingService from '@hashicorp/design-system-components/services/hds-theming';
+
+// import type {
+//   OnSetThemeCallback,
+//   OnSetThemeCallbackArgs,
+// } from '@hashicorp/design-system-components/services/hds-theming';
+
+import config from 'showcase/config/environment';
+
+export type ShwStylesheets =
+  | 'standard'
+  | 'prefers-color-scheme'
+  | 'css-selectors'
+  | 'combined-strategies';
+
+const updatePageStylesheet = (currentStylesheet: string) => {
+  let newStylesheet;
+  switch (currentStylesheet) {
+    case 'prefers-color-scheme':
+      // themed CSS where theming is applied via `@media(prefers-color-scheme)`
+      newStylesheet =
+        'assets/styles/@hashicorp/design-system-components-theming-with-prefers-color-scheme.css';
+      break;
+    case 'css-selectors':
+      // themed CSS where theming is applied via CSS selectors
+      newStylesheet =
+        'assets/styles/@hashicorp/design-system-components-theming-with-css-selectors.css';
+      break;
+    case 'combined-strategies':
+      // this is used for local testing purposes
+      newStylesheet =
+        'assets/styles/@hashicorp/design-system-components-theming-with-combined-strategies.css';
+      break;
+    default:
+      // this is the standard CSS for HDS components, without any theming
```

Comment:
```suggestion
      // this is the "standard" CSS for HDS components, without any theming
```

(just to make it stand out a bit more clearly)